runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -98,6 +98,91 @@ function M.on_refresh(err, _, ctx, _)
   return vim.NIL
 end
 
+--- @class InlayHintOptions
```

Comment:
```suggestion
--- @class vim.lsp.inlay_hints.get.opts
```

Type annotations are global, so namespacing doesn't hurt.

Reply:
changed :+1: